src/helpers/reciever.js

```diff
@@ -270,7 +274,13 @@ class Receiver {
         return response.data.body;
       })
       .catch(function (error) {
-        //console.log(error);
+        if (error.code === "ERR_BAD_REQUEST") {
+          localStorage.clear();
+          alert(
+            "Incorrect Password. This for internal use only. Please contact CloudWatch Agent team for access."
```

Comment:
Should take this wording out. This isn't a beta access thing where we'd be granting access.